modules/cli/pages/cbepctl/set-flush_param.adoc

```diff
@@ -57,13 +57,15 @@ NOTE: Changes of thresholds are NOT persistent and must be reapplied after the b
 
 alog_sleep_time, alog_task_time::
 Couchbase Server has an optimized xref:learn:buckets-memory-and-storage/memory.adoc#initialization-and-warmup[disk warmup].
-Couchbase Server pre-fetches a list of most-frequently accessed keys and fetches these documents first.
-The server runs a periodic scanner process which determines which keys are most frequently used.
-The `cbepctl flush_param` command is used to change the initial time and the interval for the process.
+An _access scanner_ is periodically run, to determine which keys have been the most frequently used: typically, the scanner writes these key to an _access log_, which can be accessed by Couchbase Server at warmup, so that the corresponding documents can be loaded first.
+(See xref:learn:buckets-memory-and-storage/memory.adoc#initialization-and-warmup[Initialization and Warmup] for further information; including cases where item-residency is at very high levels, and in consequence, no access log is created or used.)
++
+The `cbepctl flush_param` command is used to change the initial time and the interval for the access scanner.
 For example, the initial time and interval might be changed to accommodate a peak time when an application needs these keys to be quickly available.
 +
-By default, the scanner process runs once every 24 hours at 10:00 AM GMT.
-To reduce the cluster-wide impact of running this task, stagger the start time to a different value on each node in the cluster.
+By default, the access scanner runs once every 24 hours at 10:00 AM GMT.
+The scanner is highly CPU-intensive: therefore, to reduce the cluster-wide impact of running this task, its start time should be staggered to a different value on each node in the cluster.
+Note also that the scanner shares its thread pool with the Index Service; and may therefore adversely affect the performance of any simultaneous index-updates: if instances of this problem occur, the scanner should be either run with less frequency, or disabled.
```

Comment:
More specifically, the `projector` process - not sure how this is represented in the rest of the docs - calling it the Index Service here may cause confusion if people aren't running the Index Service on this node.

Reply:
"Note also that the scanner shares its thread pool with the Index Service" (or projector process).  I'm not sure I follow / agree with.  Which thread pool are we referring to? - Do we mean general CPU resources? Memcached and KV are separate processes and so don't have any direct interact other than communicating over DCP.